components/measurement/CommonSummary.js

```diff
@@ -78,6 +79,7 @@ const CommonSummary = ({
             />
             <SummaryItemBox
               label={intl.formatMessage({ id: 'Measurement.CommonSummary.Label.DateTime' })}
+              timezone={intl.formatMessage({ id: 'Measurement.CommonSummary.Label.UTC' })}
```

Comment:
@SaptakS Maybe we can keep `SummaryItemBox` a generic component and include the `timezone` label in `content` itself, like it was done for [`countryBlock`](https://github.com/ooni/explorer/pull/328/files#diff-7d296e25cd6dae8a5f0748bec6003226R74). That way `SummaryItemBox` need not know about `timezone` as a prop and we can also avoid having to do a propType check for it.